documentation.html

```diff
@@ -151,74 +151,69 @@
                     Avatar
                 </div>
                 <div class="component__header-desc">
-                    Alerts are used to draw attention of the user, to certain information like process completion/success/failure/information without interrupting the usual work flow
+                    Avatars are mainly used to display a user's profile picture on any application, they supports either image or text content.
                 </div>
                <!-- There could be multiple example illustrations for a particular component -->
                 <div class="component__example-header">
-                    Alerts without closing buttons
+                    Image Avatar
                 </div>
                 <div class="component__example-desc">
-                    Here we have 6 types of alerts, each of them serving a different purpose. Each alert box has a basic skelton (the structure, padding, and margins) and a skin (color).
-                    You can access the skelton alert box by adding an "alert" class and following it with the type of skin (color) you may require (say "alert-success").
-                    <br>
-                    To use links in alert box, use the <code class="code-in-text">.alert-link</code> utility class to provide links with colors matching to the alert box.
+                    Mantra UI provide avatars in 6 different sizes.To use avatar add <code class="code-in-text">.avatar</code> to a div and
+                    a required size indicator using any of the size classes eg. <code class="code-in-text">.avatar-xl, .avatar-lg, .avatar-md
+                     .avatar-sm, .avatar-xs</code>.
+                     <br>
+                     To make the image round and responsive add <code class="code-in-text">.img-responsive, .img-round</code>.
                 </div>
-                <div class="component__example-demo alert__example-demo-container">
-                    <div class="alert alert-primary">
-                        A simple primary alert with <a href="#" class="alert-link">an example link</a>. Give it a click if you like.
-                    </div>
-                    <div class="alert alert-secondary">
-                    A simple secondary alert—check it out!
+                <div class="component__example-demo avatar__example-demo-container">
+                    <div class="avatar avatar-xl">
+                        <img class="img-responsive img-round" loading="lazy" src="./resources/avatar-img-1.jpg" alt="avatar" srcset="">
                     </div>
-                    <div class="alert alert-success">
-                        <span><i class="fas fa-check-circle"></i></span> 
-                        A simple success alert—check it out!
+                    <div class="avatar avatar-lg">
+                        <img class="img-responsive img-round" loading="lazy" src="./resources/avatar-img-2.jpg" alt="avatar" srcset="">
```

Comment:
If you are not using srcset property, you can remove it.

Reply:
Done